lib/shared/puppet_self_service.rb

```diff
@@ -122,3 +122,25 @@ def self.filesystem_free(path)
     (stat.blocks_available.to_f / stat.blocks.to_f * 100).to_i
   end
 end
+
+def psql_return_result(sql, psql_options = '')
+  command = %(su pe-postgres --shell /bin/bash --command "cd /tmp && #{PUP_PATHS[:server_bin]}/psql #{psql_options} --command \\"#{sql}\\"")
```

Comment:
Now that EL 6 is no longer supported, we can use `runuser` instead of `su` to help avoid quoting issues.  Something like

```
runuser -u pe-postgres -- /opt/puppetlabs/server/bin/psql ${psql_options} -c "#{sql}"
```